.github/workflows/pull_request_ubuntu_build.yml

```diff
@@ -10,6 +10,7 @@ on:
       - ballerina-1.2.x
       - stage-swan-lake
       - 2201.[0-9]+.x
+      - 2201.0.3-stage-new
```

Comment:
Shall we remove these lines as they are added to test the PR builds (in the other `yml` files as well)?